packages/utilities/psammead-assets/README.md

```diff
@@ -12,15 +12,38 @@ This package provides a collection of common assets that are likely to be requir
 ```jsx
 npm install @bbc/psammead-assets --save
 ```
-
 ## Usage
 
 ```jsx
-import { BBC_BLOCKS } from '@bbc/psammead-assets/svgs';
+import { BBC_BLOCKS, news} from '@bbc/psammead-assets/svgs';
 
 import { AMP_SCRIPT } from '@bbc/psammead-assets/amp-boilerplate';
 ```
 
+## news
+
+`news` is an an object that contains an svg group, viewbox, ratio and height;
+
+<!-- prettier-ignore -->
+| Property   | Type   | Required | Default | Example                  |
+|------------|--------|----------|---------|--------------------------|
+| `group` | node | Yes | N/A | `<g fillrule="evenodd"><path d="M84.32" /></g>` |
+| `viewbox` | object | Yes | N/A | `{ height: 24, width: 167.95 }` |
+| `ratio` | number | Yes | N/A | `6.9979` |
+
+The ratio is calculated as
```

Comment:
```suggestion
The height of your SVG can be calculated using your desired height multiplied by the `ratio` value provided above.
```

I dont think we need the `const ratio = svgWidth / svgHeight;` bit :)

Reply:
It is the width not the height.